pandablocks/utils.py

```diff
@@ -107,6 +114,8 @@ def table_to_words(
             # to prevent type error, this will still work if column is another iterable
             # e.g numpy array
             packed = np.zeros((len(column), row_words), dtype=np.uint32)
+        elif not len(column):
+            column_value = np.zeros(dtype=np.uint32, shape=packed.shape[0])
```

Comment:
This line has no tests covering it

Reply:
I've changed my mind. Anything using the ioc will have to explicitly set those values on the non-defined table before submitting it. I've implemented a test for this on pandablocks-ioc.

Reply:
https://github.com/PandABlocks/PandABlocks-ioc/pull/58/commits/1a3877b95cc4c59cfcc8c80cfe22957343926f13